spyderlib/widgets/sourcecode/syntaxhighlighters.py

```diff
@@ -187,10 +191,28 @@ def set_color_scheme(self, color_scheme):
             self.color_scheme = color_scheme
         self.setup_formats()
         self.rehighlight()
-
+    

```

Comment:
There doesn't seem to be a consensus about spaces on otherwise empty lines, at least not in syntaxhighlighters.py. If anything, it is to keep those spaces.

Personally I like the indentation of empty lines to match the indentation of the next non-empty line. That is the easiest to read with 'show blank space' in my opinion, because it visually groups code that belongs together. Also it prevents the cursor from needlessly jumping around horizontally when using the up/down arrows to move.

Nevertheless, I'll change it. My editors usually add these spaces (which I like) but I remove them before committing if it is a modification of existing code. This one apparently slipped through. (I even kept the line in syntaxhighlighting.py that had just one single space until I couldn't stand it anymore a few commits ago..)